db/migrations/20230131082739_meeting_times.sql

```diff
@@ -0,0 +1,9 @@
+-- migrate:up
+CREATE TABLE meeting_times (
+  id int NOT NULL AUTO_INCREMENT,
+  meeting_time date NOT NULL,
```

Comment:
```suggestion
  time date NOT NULL,
```
테이블 이름에 정보가 담겨있기 때문에 column은 time으로만 해주셔도 될 것 같아요.

Reply:
변경하였습니다 감사합니다